### DISCUSSION TITLE
Edit password reset email

### DISCUSSION THREAD

**Discussion by achirb2** [inventree/InvenTree#3195] [Q&A] [answered]
How do I go about editing the password reset email? I would like to change the references to "example.com" to my domain name.

Thank you!

**Comment by SchrodingersGat** [comment, marked as answer]
To edit the base domain name, navigate to the "sites" section on the "admin" page:

![image](https://user-images.githubusercontent.com/10080325/173743440-5c236298-8d0c-4007-821a-285a9a1e92af.png)

**Reply by achirb2** [reply]
Thank you!

**Reply by achirb2** [reply]
I deleted example.com and added my site but I'm getting an error now and can't login with another browser or private browsing mode in firefox.

I first added the entry without added site as such:
![image](https://user-images.githubusercontent.com/5297933/174227890-c4f436a2-3f4b-4e8a-b4f0-cf91f3c6d2b4.png)


Here's the error:
![image](https://user-images.githubusercontent.com/5297933/174227713-329825cb-3d15-48d2-bdb6-e5ec94861ad0.png)


Thanks for any help you can offer!

**Reply by SchrodingersGat** [reply]
Did you actually delete and add, rather than editing the site that was already there? This might cause a problem as the code expects the SITE_ID to equal 1. If you added a new one, that might be SITE_ID 2. @matmair any thoughts?

**Reply by matmair** [reply]
@SchrodingersGat the ID could be a problem.
@achirb2 please try it without the protocol - so just domainname.tld

Also on a sidenote: I recommend to only expose the instance to the internet if you have hardned your server.

**Reply by achirb2** [reply]
> Did you actually delete and add, rather than editing the site that was already there? This might cause a problem as the code expects the SITE_ID to equal 1. If you added a new one, that might be SITE_ID 2. @matmair any thoughts?

yes this is exactly what happened. Thanks

**Reply by matmair** [reply]
@achirb2 is it working now? I am not really sure how to rectify this if not.

**Reply by achirb2** [reply]
To fix it I dropped the database and reinitialized a new one
